src/utility/MetaMode/MetaMode.cpp

```diff
@@ -0,0 +1,257 @@
+    0b11100011,0b10001110,0b00111000,0b00000000,0b01110000,0b00111000,0b11100000,
+    0b11100011,0b10001110,0b00111000,0b00000000,0b01110000,0b00111000,0b11100000,
+    0b11100011,0b10001110,0b00111000,0b00000000,0b01110000,0b00111000,0b11100000,
+    0b11100011,0b10001110,0b00111111,0b11100000,0b01110000,0b00111111,0b11100000,
+    0b11100011,0b10001110,0b00111111,0b11100000,0b01110000,0b00111111,0b11100000,
+    0b11100011,0b10001110,0b00111111,0b11100000,0b01110000,0b00111111,0b11100000,
+    0b11100011,0b10001110,0b00111000,0b00000000,0b01110000,0b00111000,0b11100000,
+    0b11100011,0b10001110,0b00111000,0b00000000,0b01110000,0b00111000,0b11100000,
+    0b11100011,0b10001110,0b00111000,0b00000000,0b01110000,0b00111000,0b11100000,
+    0b11100011,0b10001110,0b00111111,0b11100000,0b01110000,0b00111000,0b11100000,
+    0b11100011,0b10001110,0b00111111,0b11100000,0b01110000,0b00111000,0b11100000,
+    0b11100011,0b10001110,0b00111111,0b11100000,0b01110000,0b00111000,0b11100000,
+};
+
+const uint8_t textMode_w = 36;
+const uint8_t textMode_h = 12;
+const uint8_t textMode[] = {
+    textMode_w, textMode_h,
+    0b00000000,0b00000000,0b00000000,0b00000000,0b00000000,
+    0b01111111,0b11100111,0b11100111,0b10000111,0b11100000,
+    0b01111111,0b11100111,0b11100111,0b10000111,0b11100000,
+    0b01100110,0b01100110,0b01100110,0b01100110,0b00000000,
+    0b01100110,0b01100110,0b01100110,0b01100110,0b00000000,
+    0b01100110,0b01100110,0b01100110,0b01100111,0b11100000,
+    0b01100110,0b01100110,0b01100110,0b01100111,0b11100000,
+    0b01100110,0b01100110,0b01100110,0b01100110,0b00000000,
+    0b01100110,0b01100110,0b01100110,0b01100110,0b00000000,
+    0b01100110,0b01100111,0b11100111,0b10000111,0b11100000,
+    0b01100110,0b01100111,0b11100111,0b10000111,0b11100000,
+    0b00000000,0b00000000,0b00000000,0b00000000,0b00000000,
+};
+
+const uint8_t NUMBER_OF_COLORS_LINE_PATTERN = 8;
+const Color line_pattern[NUMBER_OF_COLORS_LINE_PATTERN] = {  // Color of the bottom and top bars
+    WHITE, WHITE, WHITE, WHITE, ORANGE, ORANGE, ORANGE, ORANGE};
+
+const Color rects_pattern[4] = {  // Colors of the sliding rectangles during launch animation
+    WHITE, BEIGE, ORANGE, BROWN};
+
+uint8_t current_pattern = 0;
+
+bool anim_running = false;
+int16_t anim_timer = 0;
+const uint8_t anim_rect_time_max = 10;  // Time per sliding rect
+
+const uint8_t anim_h_start = 18;  // Height of the sliding rects when the start in the center
+const uint8_t anim_h_end = 2;     // Their height at the last frame
+const uint8_t anim_interval = 7;  // Frames between 2 sliding rectangles
+
+uint8_t loading_timer = 0;
+const uint8_t loading_time_max = 40;  // Time in frames you have to hold HOME and MENU
+
+bool anim_unhandled_running = false;
+const uint8_t anim_unhandled_time_max = 8;
+uint8_t anim_unhandled_timer = 0;
+
+bool MetaMode::isActive() {
+	handled = true;
+	return active;
+};
+
+bool MetaMode::is_using_button_home() {
```

Comment:
Please try sticking with one variable naming format, so this would be `MetaMode::isUsingHomeButton` instead of using the underscores

Reply:
Oh my bad, I tried to use camelCase because it is used everywhere in the library, but I guess my habit with underscores kicked in hehe